SWIG/piecewiseyieldcurve.i

```diff
@@ -84,7 +84,7 @@ export_piecewise_curve(PiecewiseLinearForward,ForwardRate,Linear);
 export_piecewise_curve(PiecewiseLinearZero,ZeroYield,Linear);
 export_piecewise_curve(PiecewiseCubicZero,ZeroYield,Cubic);
 export_piecewise_curve(PiecewiseLogCubicDiscount,Discount,MonotonicLogCubic);
-
+export_piecewise_curve(PiecewiseNonMonotonicLogCubicDiscount,Discount,LogCubic);
```

Comment:
This would use `SplineCubic` (see above)

Reply:
The whole reason I was going for this change was to test out if the monotonic constraint on the interp is what was causing some issues for me in the extreme front end of the curve when bootstrapping.  I'm noticing that the o/n forwards on the 3mL curve are concave from T0 (even when using proper jumps), so relaxing the monotonic constraint might help.  If not though, then I will have to resort to marking the o/n forwards explicitly which is not ideal, but a solution nonetheless.